guides/learning_guide.rst

```diff
@@ -0,0 +1,1094 @@
+   second level instances each managing 32 cores, schedule the workload 45x faster than the traditional workload
+   manager (red) and scales well with the number of cores. Finally, three levels of Flux instances (green) with each third-level
+   instance managing a single core, maximizes throughput at 60x faster than traditional techniques (red) and scales
+   excellently with the number of cores.
+   
+   
+Scalable Graph-Based Scheduling Techniques
+==========================================
+
+Once instantiated, Flux effectively carries out many complicated operations
+ranging from monitoring the health of resources, to enqueuing submitted tasks,
+to scheduling and executing these tasks. Flux uses a graph-based approach to
+scheduling as described earlier. Flux's scheduler component, called Fluxion, is
+represented in Figure 11. Furing Flux instance initialization, Fluxion first populates
+an in-memory resource graph store (A) comprising vertices that represent the HPC
+system's various compute resources and edges that represent the relationships
+among those resources. The initialization process also includes the selection of the
+graph resource's representation granularity and traversal type if users decided
+to use non-default. Once initialization is complete, Fluxion is ready to receive the
+jobs' resource requests from Flux's core framework.
+
+Flux first constructs a job's resource request in the form of an abstract resource
+request graph (B). The abstract request graph generally specifies the job's resource
+requirements in terms of both node-local resources (e.g., amount of compute cores
+and memory to be used) and higher level or even global resources (e.g., compute
+racks, network switches, power, parallel filesystem bandwidth). The abstract request
+graph then becomes the input for the selected graph traverser (A) to find its best-
+matching resource vertices and edges. The traverser “walks” the concrete resource
+graph store in this pre-defined walking order and matches the abstract request
+graph to the concrete resource graph.
+
+.. figure:: images/figure-11.png
+   :alt: Fluxion scheduling rpocess within Flux
+   :align: center
+
+   Figure 11: Fluxion, Flux's scheduler component, automatically performs multiple steps when a Flux instance is initialized. 
+   See text for a description of each lettered process.
+
+
+
+As shown at (C), the best-matching criteria is determined by the match policy within
+Flux's traverser. The policy is invoked every time the traverser visits a vertex. The
+policy then evaluates how well a given resource vertex matches with the abstract
+request graph and scores it accordinly. Flux's resource model must also efficiently
+keep track of the status changes of resources over time in order to support various
+queuing and backfilling policies common to HPC job scheduling (e.g., EASY and
+conservative backfilling policies). Thus, the model integrates a highly
```

Comment:
While I guess EASY was originally a project, it seems many people refer to it as a backfill policy or heuristic these days:
e.g.

https://jsspp.org/papers17/paper_5.pdf

> EASY-Backfilling is a popular scheduling heuristic for allocating jobs in large scale High Performance Computing platforms

https://ieeexplore.ieee.org/document/4629218

> The most frequently used algorithm, EASY-backfilling, was selected for a performance evaluation by scheduling static workloads of parallel jobs on a computer cluster.

So I don't think it would be appropriate to call it a project here. "EASY policy" is probably correct.

Reply:
hmmm, alternatively would "EASY or other conservative backfill policies" be better?  Although I dislike changing the text too much.

Reply:
I'm not sure that makes sense, "conservative" is another type of backfill policy. Would it be more clear if it was reworded to state "e.g. backfill policies such as EASY and conservative"?

Reply:
Oh I didn't realize that "conservative" was the name of a policy.  Yeah your adjustment sounds better.